tck/src/main/java/org/eclipse/microprofile/openapi/tck/PetStoreAppTest.java

```diff
@@ -315,9 +314,14 @@ public void testExtensionPlacement(String type) {
 
         vr.body(opPath, hasEntry(equalTo("x-operation-ext"), equalTo("test-operation-ext")));
 
-        vr.body(opPath + ".responses.'200'", not(hasProperty("'x-operation-ext")));
+        vr.body(opPath + ".responses", hasEntry(equalTo("x-responses-ext"), equalTo("test-responses-ext")));
+        vr.body(opPath + ".responses.'200'", not(hasKey("'x-operation-ext")));
         vr.body(opPath + ".responses.'200'", hasEntry(equalTo("x-response-ext"), equalTo("test-response-ext")));
-        vr.body(opPath + ".responses.'500'", not(hasProperty("'x-operation-ext")));
+        vr.body(opPath + ".responses.'500'", not(hasKey("'x-operation-ext")));
```

Comment:
Lines 318 and 320 have an extra `'` character (`'x-operation-ext` instead of `x-operation-ext`).

Since they're negative tests, I'd expect them to pass anyway, but I don't think they're testing what they're meant to be.

Reply:
Good catch - fixed them.